### ISSUE TITLE
Why can't i move?

### ISSUE THREAD

**Issue by TRCGaming** [OgarProject/Ogar#377]
I connected to my private server but i can't move... Pls help ASAP!

**Comment by BielSAA** [comment]
Give us more informations, are you using a modded source of Ogar ? Have you paused the game ? (to unpause type "pause" withou quotes in the console and press enter) Are you using official agario client ?

**Comment by TRCGaming** [comment]
I modded some things and i didn't pause the game, i can W and split bu i'm in 1 spot.

**Comment by TRCGaming** [comment]
And now I can't even connect.

**Comment by MastaCoder** [comment]
"I modded some things" ~ Might that be the issue?

**Comment by TRCGaming** [comment]
I saw another guy do the same thing but he didn't mod, and he also couldn't move...

**Comment by Luka967** [comment]
Can you leave us a duplicate of your code on Github? I would like to see it.

**Comment by TRCGaming** [comment]
Find TK-Turkey
main_out.js?v=39:3 Connecting to ws://127.0.0.1:443
main_out.js?v=39:3 WebSocket connection to 'ws://127.0.0.1:443/' failed: Error during WebSocket handshake: Unexpected response code: 404
main_out.js?v=39:3 socket error
main_out.js?v=39:3

**Comment by MastaCoder** [comment]
That means the server isn't open on the port. Are you sure it's running and nothing else is in the way?

**Comment by TRCGaming** [comment]
Pretty sure, let me try again

**Comment by TRCGaming** [comment]
I connected, but i still can't move. Can you watch the video i watched and tell me what the problem is if you know? Here is his channel: Hero ⚡ 
You know him, agar.io hero? Anyways thanks

**Comment by vram4** [comment]
You need to make sure you are using the latest version of ogar.

**Comment by Tarzam3** [comment]
I have this problem too... I cant move.
